src/02_spec/07_modules/dem_uart/systemif.rst

```diff
@@ -0,0 +1,53 @@
+System Interface
+----------------
+
+The following signals are required from a bus in order to connect to the DEM.
```

Comment:
Please give the specification for these signals. What addresses are allowed? What is the timing relationship between bus_req and bus_wdata, etc?